### PR TITLE
Add leading message. #1

### PR DESCRIPTION
**Pull request by mairamdnor980526**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
   <title>Welcome to Potionfy</title>
 </head>
 <body>
-  
+  <h1>Enjoy your stay!</h1>
```
Comment (mairamdnor980526) [inline index.html:10]: Could you remove the `Enjoy your stay.` string and make the `Welcome to Potionfy!` string red?

Comment (mairamdnor98) [inline index.html:10]: Could you remove the `Enjoy your stay!` string and make the `Welcome to Potionfy!` string red?

Comment (mairamdnor98) [inline index.html:10]: Could you remove the `Enjoy your stay!` string and make the `Welcome to Potionfy!` string red?

**Review by mairamdnor98 — COMMENTED** [review]
Please, change the code as per my comments on the review.

File: index.html (hunk 1 above)
Comment (mairamdnor980526) [reply index.html:10]: Sure, can do!

**Review by mairamdnor98 — COMMENTED** [review]
Please, change the code as per my comments on the review.

**Pull request merged**